Cargo.lock

```diff
@@ -2435,6 +2483,16 @@ dependencies = [
  "uuid",
 ]
 
+[[package]]
+name = "names"
+version = "0.12.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "10a8690bf09abf659851e58cd666c3d37ac6af07c2bd7a9e332cfba471715775"
+dependencies = [
+ "clap 3.0.0-beta.2",
```

Comment:
Is there a feature flag on `names` we could use to remove the clap dependency? It's a large one.

Reply:
it sure does.